Scripts/operations.py

```diff
@@ -11,7 +11,7 @@
 import time
 import itertools
 from datetime import datetime
-from rapidfuzz import fuzz
+from rapidfuzz.distance import Indel
```

Comment:
Both `rapidfuzz.fuzz.ratio` and `Levenshtein.ratio` work with a normalized Indel similarity under the hood. This is basically the Levenshtein distance with the following weights: (deletions=1, insertions=1, substitutions=1). Is this actually what is intended, given the config talks about the Levenshtein distance?